src/sequential.jl

```diff
@@ -53,6 +53,8 @@ function reapply(s::SequentialTransform, table, cache)
   current
 end
 
+Base.:(==)(s1::SequentialTransform, s2::SequentialTransform) = s1.transforms == s2.transforms
```

Comment:
I find it surprising that Julia doesn't add this method by default. Thought that structs were compared field-by-field as a fallback.